loom/graphics/gfxVectorGraphics.cpp

```diff
@@ -170,7 +170,6 @@ void VectorGraphics::endFill() {
 void VectorGraphics::moveTo(float x, float y) {
     auto path = getPath();
     path->moveTo(x, y);
-    inflateBounds(Loom2D::Rectangle(x, y, 0, 0));
```

Comment:
What do you mean `-50, -50, 150, 150`? I beilive it works as expected, drawing a line from `-50, -50` to `100, 100` with the proper bounds being set.